<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentMainWindow</class>
 <widget class="QMainWindow" name="StudentMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	border-radius:3px;
}
QDialog{
background-color:rgb(230, 230, 230);
}
QLineEdit
{
    font: 25 9pt &quot;Microsoft JhengHei&quot;;
    background:white;
    padding-left:4px ;
    padding-top:4px ;
    border-bottom-left-radius:4px;
    border-bottom-right-radius:4px;
    border: 1.2px solid rgb(189 , 189, 189);
    border-top:transparent;
    
}
QLineEdit:hover
{
    font: 25 9pt &quot;Microsoft JhengHei&quot;;
    padding-top:0px ;
    border: 1px solid rgb(21 , 131 , 221);
}
QLineEdit:focus
{
    font: 25 9pt &quot;Microsoft JhengHei&quot;;
    padding-top:0px ;
    border: 1px solid rgb(21 , 131 , 221);
}

QPushButton
{
	font: 25 9pt &quot;Microsoft JhengHei&quot;;
    color:white;
    background-color:rgb(75,164,242);
    border-radius:3px;
}

QPushButton:hover
{
	font: 25 10pt &quot;Microsoft JhengHei&quot;;
    color:white;
    background-color:rgb(44 , 137 , 255);
	border-radius:5px;
}

QPushButton:pressed
{
	font: 25 11pt &quot;Microsoft JhengHei&quot;;
	border-radius:13px;
    color:white;
    background-color:rgb(14 , 135 , 228);
    padding-left:5px;
    padding-top:5px;
}

QPushButton:default
{
	font: 25 9pt &quot;Microsoft JhengHei&quot;;
	border-radius:3px;
    color:white;
    background-color:rgb(44 , 137 , 255);
    padding-left:3px;
    padding-top:3px;
    border: 2px solid rgb(17 , 120 , 221);
}

QPushButton:!enabled
{
	font: 25 9pt &quot;Microsoft JhengHei&quot;;
    color:white;
    background-color:rgb(200 , 200 , 200);
	border-radius:5px;
}
QRadioButton
{
    font: 25 9pt &quot;Microsoft JhengHei&quot;;
}
QLabel#TopImage
{
	border-radius:90px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QPushButton" name="btnExitClass">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Quit Class</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="edtIP">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
      <property name="placeholderText">
       <string>请输入教师端IP</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1920</width>
           <height>1080</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>1920</width>
           <height>1080</height>
          </size>
         </property>
         <widget class="QLabel" name="lblSharedScreen">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1920</width>
            <height>1080</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btnChat">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Chat</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnEnterClass">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Enter Class</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="lblAttention">
      <property name="font">
       <font>
        <family>Microsoft JhengHei</family>
       </font>
      </property>
      <property name="text">
       <string>Attention: --</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
